### PR TITLE
[GUI] A more accurate tooltip for highlights reconstruction module

### PR DESCRIPTION
**Pull request by victoryforce**
This module only works with RAW and sRAW images, which are not monochrome images. So it won't work with color JPEG images, for example. But at the same time, the tooltip to the text "not applicable" in the body of the module contained the explanation "no highlights reconstruction for monochrome images", which is incorrect in the case of color JPEG.

Additionally, the "not applicable" text is not a widget label, so it doesn't need to be left-aligned, it looks better centered.

### CODE REVIEW

File: src/iop/highlights.c [hunk 1]
```diff
@@ -2621,8 +2622,9 @@ void gui_init(struct dt_iop_module_t *self)
   gtk_widget_set_tooltip_text(g->scales, _("increase to correct larger clipped areas.\n"
                                            "large values bring huge performance penalties"));
 
-  GtkWidget *monochromes = dt_ui_label_new(_("not applicable"));
-  gtk_widget_set_tooltip_text(monochromes, _("no highlights reconstruction for monochrome images"));
+  GtkWidget *monochromes = gtk_label_new(_("not applicable"));
+  gtk_widget_set_halign(monochromes, GTK_ALIGN_CENTER);
```
Comment (TurboGit) [inline src/iop/highlights.c:2626]: I'm not sure about this. I'm not against but if we go this way we probably want to do that for all other modules with the same kind of restrictions. So I'll prefer having this out of this PR and create a new PR for the alignment, but not that some mode have long restriction description like `raw chromatic aberrations`:

image

At the end I'm not sure the center alignment will work fine.

Comment (victoryforce) [reply src/iop/highlights.c]: OK. If alignment needs discussion, I've removed the corresponding commit from PR. Now PR only corrects the inaccuracy of the tooltip text. But I've also added renames to the relevant elements in the source code to more accurately reflect what they're for.

**Comment by TurboGit** [comment]
> This module only works with RAW and sRAW images, which are not monochrome images.

Except for Leica monochrome DNG which are RAW and not supported by highlight module. @jenshannoschwalm: Do you confirm that?

**Comment by victoryforce** [comment]
> > This module only works with RAW and sRAW images, which are not monochrome images.
> 
> Except for Leica monochrome DNG which are RAW and not supported by highlight module.

This is exactly what I wrote.

**Comment by victoryforce** [comment]
@TurboGit Ah, I see what you mean. My new tooltip, while more accurate in most cases, will not be accurate for Leica monochrome DNG. That is, it is better to make it more verbose and write "this module only works with non-monochrome RAW and sRAW"

**Comment by TurboGit** [comment]
> That is, it is better to make it more verbose and write "this module only works with non-monochrome RAW and sRAW"

Yep, exactly my point.

**Pull request merged**